fleet/lib/kube-prometheus-stack-pre/externalsecret-grafana-keycloak-credentials.yaml

```diff
@@ -18,7 +18,7 @@ spec:
   - remoteRef:
       key: grafana-keycloak-credentials
       property: hostname
-    secretKey: keycloak_url
+    secretKey: url
```

Comment:
I looked but I can't figure it out. Where is this used?

Reply:
I think we should either update `    auth.generic_oauth` to use this value or remove it from the secret.